book/website/collaboration/inclusive-events/inclusive-events-planning.md

```diff
@@ -1,31 +1,362 @@
+
+(cl-inclusive-events-planning-honorarium)=
+#### Honorarium and Bursaries 
+
+Depending on the size of the budget and structure, creating a policy and the budget for an Accessibility Fund can help to make your event inclusive for all participants, regardless of their needs and circumstances. 
+This is a Fund that can be applied to cover additional costs that would support people with lived experience attending and engaging in your event. These additional costs could be: 
+- additional accommodation for people who may need to travel to an event the day prior, or stay longer, if attending in person due to health conditions.
+- any travel costs or support for those with caring responsibilities to cover the cost of a support worker, childcare, or allowing their child to travel with them.
+- honorarium for members of the public with lived experience to attend sessions. 
+- honorariums for speakers, facilitators, and note-takers if they are needed for your event.
+- support for accessibility requirements such as a Sign Language Interpreter.
+- reimbursements for accessible transport (for example, taxis can often be more expensive than public transport but may be required depending on the location of your event).
+
+It will be important to create a clear and understandable policy for what activities and costs can be covered by an Accessibility fund. 
+Honorariums and Reimbursements may have separate policies and timelines, and it is important to be as transparent as possible in order to support people with lived experience engaging in your event. 
+It is advisable to highlight who is responsible for payments, how any reimbursement procedures will work and their expected timeline, and clear guidance on what expenses will be covered. 
+
+Please ensure that all documentation and communication about honorarium and reimbursements are written in an Easy Read format following plain language guidance and if possible, provide clear pathways on how to submit documents. 
+People with lived experience may not have a printer in their homes and may share a computer or smartphone to take part in events. 
+Where possible, please be mindful of the technology and process barriers that people may face and make processes as clear as possible.
+
+```{admonition} Case Study
+:class: tip
+A community event attended by people with vision impairments requested a support person to type and guide them through a reimbursement form. 
+While this may not be a requirement for all types of events, consider both the budget, communication, and the GDPR requirements that would be needed for a member of staff to support attendees with completing the forms. 
+```
+
+(cl-inclusive-events-planning-risk-assessment)=
+### Risk Assessment
+
+Many organizations will require a risk assessment as part of planning an event and it is critical to consider the additional risks that may be unique to the attendees of your specific event. 
```

Comment:
maybe also good to point out that even if the org itself doesn't require it that it can be a good exercise to think through potential risks?